cli/source/utils/run-command.ts

```diff
@@ -5,6 +5,7 @@ import {
 } from 'child_process';
 
 import CircularBuffer from './circularBuffer.js';
+
 import { LogSource, READY_MESSAGES } from '../constants.js';
 
```

Comment:
Remove this extra line. I don't think this is need. Was is added by the linter?
```suggestion
import { LogSource, READY_MESSAGES } from '../constants.js';
```

Reply:
I'd recommend not blocking the PR over this specific comment since the linter should ideally deal with it.

Reply:
Got it. Will look into the linter problems